Aptoide/aptoide-client-v8
# [feature/ASV-803] Bonus Appc Flair to Editorial

**What does this PR do?**

   Implements the new bonus appc flair (e.g. 25% bonus) to editorial cards.

**Database changed?**

   No

**How should this be manually tested?**

  Check with Charles editorial cards that have the "flair" field set to ""appc-bonus-25". For those, the new bonus appc flair should show in the card. The rest of the cards should be the same.

**What are the relevant tickets?**

  [MOB-803](https://aptoide.atlassian.net/browse/MOB-803)

**Code Review Checklist**
